codegen/src/main/java/io/improbable/keanu/codegen/python/DocString.java

```diff
@@ -0,0 +1,44 @@
+package io.improbable.keanu.codegen.python;
+
+import java.util.Map;
+
+class DocString {
+    private static final String THREE_QUOTES = "\"\"\"";
+    private static final String NEW_LINE_TAB = "\n    ";
+
+    private String comment;
+    private Map<String, String> params;
```

Comment:
Make `comment` and `params` final

Reply:
I don't really like the naming either - can we go for `description` or `documentation`? And what's `params`? Guessing it stores the name and the description - `parameterNameAndDescription`?

Reply:
So the reason for using `comment` is because that is what it is called in the standard Doclet API but I agree that we can do better. Will update.